<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowBox</class>
 <widget class="QWidget" name="ShowBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnMinimize">
     <property name="text">
      <string>Minimize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMaximize">
     <property name="text">
      <string>Maximize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnShowNormal">
     <property name="text">
      <string>Show normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbKeepOnTop">
        <property name="text">
         <string>Keep on top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDontKeepOnTop">
        <property name="text">
         <string>Don't keep on top</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinX">
       <property name="minimum">
        <number>-10000</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinHeight">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>W:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinY">
       <property name="minimum">
        <number>-10000</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>H:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinWidth">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnMinimize</sender>
   <signal>clicked()</signal>
   <receiver>ShowBox</receiver>
   <slot>minimizeTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMaximize</sender>
   <signal>clicked()</signal>
   <receiver>ShowBox</receiver>
   <slot>maximizeTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShowNormal</sender>
   <signal>clicked()</signal>
   <receiver>ShowBox</receiver>
   <slot>showNormalTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ShowBox</receiver>
   <slot>updatePositionTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ShowBox</receiver>
   <slot>updatePositionTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ShowBox</receiver>
   <slot>updatePositionTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ShowBox</receiver>
   <slot>updatePositionTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbKeepOnTop</sender>
   <signal>toggled(bool)</signal>
   <receiver>ShowBox</receiver>
   <slot>updateTopmost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>minimizeTarget()</slot>
  <slot>maximizeTarget()</slot>
  <slot>showNormalTarget()</slot>
  <slot>updatePositionTarget()</slot>
  <slot>updateTopmost()</slot>
 </slots>
</ui>
